frontend/packages/helm-plugin/src/components/details-page/resources/HelmReleaseResourcesRow.tsx

```diff
@@ -37,16 +37,16 @@ const HelmReleaseResourcesRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ o
       <TableData className={tableColumnClasses.name}>
         <ResourceLink
           kind={kind}
-          name={resource.metadata.name}
-          namespace={resource.metadata.namespace}
+          name={resource.metadata?.name}
+          namespace={resource.metadata?.namespace}
```

Comment:
Again, this will be a broken link if these values aren't set. Should we handle that?